examples/benches/clock.rs

```diff
@@ -0,0 +1,18 @@
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
```

Comment:
this feel out of place.

Reply:
The code is in the middle of the development process. PR is a draft. I was half-way thro. Don't look at it until it's not a draft.